platform/core/src/extensions/MODULE_TYPES.js

```diff
@@ -1,6 +1,7 @@
 export default {
   COMMANDS: 'commandsModule',
   CUSTOMIZATION: 'customizationModule',
+  STATE_SYNC: 'stateSyncModule',
```

Comment:
Are there any examples of this? If not why include it? If we are going to include it then I would strongly suggest we include the *.md documentation file that describes this.

Reply:
It just prevents throwing an exception.  No, there isn't anything that uses it, but you need to list it here.

Reply:
I am confused. Why does this need to be here?

Reply:
How/when/why would an exception be thrown if it were not here?